src/foam/u2/detail/RowPropertyView.js

```diff
@@ -18,6 +18,7 @@ foam.CLASS({
     ^ {
       display: flex;
       justify-content: space-between;
+      margin: 5px;
```

Comment:
Why do we need this?

Reply:
@sarthak-marwaha This is for the 'From', 'Amount', and 'Frequency' in the section of the 'Schedule' on the 'Review your Subscription' page.  This makes the 'Schedule' same left alignment as Limit(s) 
<img width="445" alt="image" src="https://user-images.githubusercontent.com/55885868/186736661-9cbad6e9-a16e-4afa-8c34-778f5ae6f222.png">

Reply:
We should reduce the padding/margin in CollapseBorder Title rather than adding more margin here since it is already a narrow view